analysis_options.yaml

```diff
@@ -1,5 +1,12 @@
-include: package:flutter_lints/flutter.yaml
-
+include: package:very_good_analysis/analysis_options.6.0.0.yaml
```

Comment:
Might be, but it does enforce some good styles too. I rather keep it in for development and switch back before release. This will ensure some best practices.

Reply:
The best practices defined by Very Good Ventures, not by Flutter.
I think in the future, when another dev need implement something here, is much probably that know the Flutter recommended standards than know the Very Good Ventures standards.